Ratnesh/week 2 assignment 1 by ratnesh/addition 2d matrix.c

```diff
@@ -0,0 +1,32 @@
+#include <stdio.h>
+ 
+int main()
+{
+   int m, n, c, d, first[10][10], second[10][10], sum[10][10];
+ 
+   printf("Enter the number of rows and columns of matrix\n");
+   scanf("%d%d", &m, &n);
+   printf("Enter the elements of first matrix\n");
+ 
+   for (c = 0; c < m; c++)
```

Comment:
Always better to use the scope {}. for better readability of code.

Reply:
ok, I accept for readability, but sometimes it increases the time complexity of a program.